src/test/groovy/com/piekoszek/nowaksharedrent/auth/AuthServiceTest.groovy

```diff
@@ -42,4 +55,26 @@ class AuthServiceTest extends Specification {
         authService.createAccount(firstAccount)
         !authService.createAccount(secondAccount)
     }
+
+    def "Saving an account with wrong email" () {
```

Comment:
This validation tests are more Controller than Service tests.

Reply:
Account class I would say ;) however they should stay in the same package anyway

Reply:
Hmm I think validation doesn't work, try it from Postman.